track_analyzing/track_analyzer/track_analyzer.cpp

```diff
@@ -0,0 +1,123 @@
+#include "track_analyzing/exceptions.hpp"
+#include "track_analyzing/track.hpp"
+
+#include "indexer/classificator.hpp"
+#include "indexer/classificator_loader.hpp"
+
+#include "3party/gflags/src/gflags/gflags.h"
+
+using namespace std;
+using namespace track_analyzing;
+
+namespace
+{
+#define DEFINE_string_ext(name, value, description)                           \
+  DEFINE_string(name, value, description);                                    \
+                                                                              \
+  string const & Demand_##name()                                              \
```

Comment:
The word "demand" is misused here from my point of view. I suggest you to replace it by "Checked", i.e. "Checked_in(), Checked_mwm(), ...". @mpimenov WDYT?